lib/blocks.js

```diff
@@ -800,14 +800,18 @@ var finalize = function(block, lineNumber) {
 // Walk through a block & children recursively, parsing string content
 // into inline content where appropriate.
 var processInlines = function(block) {
-    var node, event, t;
+    var event;
     var walker = block.walker();
     this.inlineParser.refmap = this.refmap;
     this.inlineParser.options = this.options;
     while ((event = walker.next())) {
-        node = event.node;
-        t = node.type;
-        if (!event.entering && (t === 'paragraph' || t === 'heading')) {
+        if (event.entering) {
+          continue
+        }
+
+        var node = event.node;
+        var t = node.type;
+        if (t === 'paragraph' || t === 'heading') {
             this.inlineParser.parse(node);
```

Comment:
What does this gain us?

Reply:
This change can reduce times of variable t's assignment for a little performance. But I made a mistake, the declaration `var t` and `var node` should before while loop, I will revert it later, sorry.